app/javascript/packs/pages/ProviderPage.jsx

```diff
@@ -87,57 +122,70 @@ export default function ProviderPage(props) {
+            {neighborhood ? `${neighborhood} / ${city}, ${state}` : `${city}, ${state}`}
+          </Typography>
+          <Typography component="h3" variant="h6" align="center" gutterBottom className={classes.role}>
+            {role}
+          </Typography>
+          <Typography component="h3" variant="h6" align="center" className={classes.facility}>
+            {facility}
+          </Typography>
+          <Typography component="h4" variant="h5" align="center" gutterBottom className={classes.sectionHeaders}>
+            Needs
+          </Typography>
+          { requests.length === 0 ?
+            "No needs at this time."
+            :
+            <List>
+              {requests.map((request, index) => {
+                return (
+                  <ListItem className={[classes.listItem, request.satisfied ? classes.faded : ""].join(" ")} key={index}>
+                    {request.satisfied ? `${request.type} - satisfied` : request.type}
+                  </ListItem>
+                )
+              })}
+            </List>
+          }
+          <Typography component="h4" variant="h5" align="center" color="textPrimary" gutterBottom className={classes.sectionHeaders}>
+            Details
+          </Typography>
           <Typography component="p" variant="h6" align="center" color="textPrimary" gutterBottom className={classes.description}>
             {description}
           </Typography>
+          { active ? (
+            <Button variant="contained" className={classes.button} href={`/volunteer-signup?provider=${id}`}>
+              Offer Help
+            </Button>
+          ) : (
+            <Typography component="h1" align="center" className={classes.complete} gutterBottom>
+              The provider has marked this request as complete.
+            </Typography>
+          )}
         </Container>
-        { active ? (
-          <Button variant="contained" className={classes.button} href={`/volunteer-signup?provider=${id}`}>
-            Offer Help
-          </Button>
-        ) : (
-          <Typography component="h1" align="center" className={classes.complete} gutterBottom>
-            The provider has marked this request as complete.
-          </Typography>
-        )}
-      </Container>
-    </Paper>
+      <Box className={classes.requestHeader}>
+        <Typography component="p" variant="h6" align="center">
+          Find more care providers to help <Link to="/browse" className={classes.browseLink}>here.</Link>
+        </Typography>
+      </Box>
+      <Box className={classes.footer}>
+        <Typography component="p" align="center" gutterBottom>
+          <b>Need help editing or removing a request?</b>
+        </Typography>
+        <Typography component="p" align="center" gutterBottom>
+          Please email <a href="mailto:covidcarenetwork@gmail.com">covidcarenetwork@gmail.com</a> to report requests that should be updated or removed.
```

Comment:
Please change to support@feedadoc.zendesk.com (right @mavenkeith ?)

Reply:
Changed 👍